### PR TITLE
Refactored import_playbook tests

### PR DESCRIPTION
**Pull request by ssbarnea**
Updated our import_playbook tests to match Ansible implementation of import_playbook and to avoid hitting Ansible syntax check failures with our own testing playbooks.

Ansible can only import playbooks at top level, not via tasks thus our testing was incorrect. 

Needed-By: #950

### CODE REVIEW

File: test/TestImportPlaybook.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+"""Test ability to import playbooks."""
+from ansiblelint.runner import Runner
+
+
+def test_task_hook_import_playbook(default_rules_collection):
+    """Assures import_playbook includes are recognized."""
+    playbook_path = (
+        'test/playbook-import/playbook_parent.yml'.
+        format_map(locals())
```
Comment (awcrosby) [inline test/TestImportPlaybook.py:9]: what is the purpose of this since there are no variable placeholders in the path string?

Comment (ssbarnea) [reply test/TestImportPlaybook.py]: Thanks. Fixed it already, leftover from previous version.

File: test/playbook-import/playbook_imported.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@
   connection: local
   gather_facts: no
   tasks:
-  - command: echo "no name"  # noqa 502
+  - command: echo "no name"  # should generate 502
```
Comment (rafaelfolco) [inline test/playbook-import/playbook_imported.yml:6]: 502 expected check

File: test/playbook-include/playbook_post.yml [hunk 3]
```diff
@@ -1,8 +0,0 @@
----
-- hosts: localhost
-  tasks:
-    - command: echo "no name"  # noqa 502
-  post_tasks:
```
Comment (rafaelfolco) [inline test/playbook-include/playbook_post.yml:5]: no longer importing as post_tasks, hope this does not lose coverage

File: test/playbook-include/playbook_pre.yml [hunk 4]
```diff
@@ -1,8 +0,0 @@
----
-- hosts: localhost
-  tasks:
-    - command: echo "no name"  # noqa 502
-  pre_tasks:
```
Comment (rafaelfolco) [inline test/playbook-include/playbook_pre.yml:5]: ditto

File: test/playbook-include/playbook_post.yml (hunk 3 above)
Comment (ssbarnea) [reply test/playbook-include/playbook_post.yml:5]: Yeah. This is what I wanted.

**Pull request merged**